kuri_camera/src/uds_to_tcp.cpp

```diff
@@ -16,6 +16,14 @@ int main(int argc, char **arcv) {
     return -1;
   }
 
+  // Sleep for some time to ensure that the masumx daemon has fully started
+  int sleep_secs;
+  if (!ros::param::get(ros::this_node::getName()+"/sleep_secs", sleep_secs))
```

Comment:
More common to see this with a `~/<param name>`, though I think it's functionally identical

Reply:
Good to know, thanks!